### DISCUSSION TITLE
HRRR DEM?

### DISCUSSION THREAD

**Discussion by zflamig** [NOAA-GSL/GSL-Regional-Model-Forum#41] [Q&A] [answered]
Is the DEM used in HRRR available for download somewhere? This would be extremely useful for matching observations to the appropriate HRRR grid cell.

Thanks!

**Comment by blaylockbk** [comment, marked as answer]
The GRIB parameter [`HGT:surface`](https://herbie.readthedocs.io/en/latest/user_guide/_bonus_notebooks/model_terrain.html) in the `sfc` files is the terrain height, isn't it?

**Reply by EricJames-NOAA** [reply]
Yes, HGT:surface is the terrain height!

**Reply by zflamig** [reply]
Thanks! I wasn't sure that this was unmodified.
